libs/ui/src/components/NeoSkeleton/NeoSkeleton.scss

```diff
@@ -13,11 +13,20 @@ $grey-lightest: hsl(0, 0%, 93%);
 $skeleton-margin: 0.25rem;
 $base-border-radius: 4px;
 .neo-skeleton {
-
   &-item {
     &.no-margin {
       margin: 0 !important;
     }
+
+    background-size: 400% 100%!important;
```

Comment:
why is this needed?

Reply:
it is getting overridden and we need it for animation